packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -26,139 +32,107 @@ const BottomWidgetsMobile = ({ tick, digits, setTick, setDigits }) => {
     return null;
 };
 
-class Trade extends React.Component {
-    state = {
-        digits: [],
-        tick: {},
-        try_synthetic_indices: false,
-    };
-
-    componentDidMount() {
-        this.props.onMount();
-    }
+const Trade = props => {
+    const [digits, setDigits] = React.useState([]);
+    const [tick, setTick] = React.useState({});
+    const [try_synthetic_indices, setTrySyntheticIndices] = React.useState(false);
+    const [is_digits_widget_active, setIsDigitsWidgetActive] = React.useState(false);
 
-    componentWillUnmount() {
-        this.props.onUnmount();
-    }
+    React.useEffect(() => {
+        props.onMount();
+        return () => props.onUnmount();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
is there a better way around using these `eslint-disable-next-line` comments?

Reply:
done